docker-compose.yml

```diff
@@ -0,0 +1,211 @@
+version: '3.8'
+
+services:
+  angular-build:
+    build:
+      context: .
+      dockerfile: ./docker/Dockerfile
+    container_name: oppia-angular-build
+    volumes:
+      - .:/app/oppia
+      - shared_node_modules:/app/oppia/node_modules
+      - shared_third_party:/app/oppia/third_party
+      - shared_proto_files:/app/oppia/proto_files
+      - shared_js_ts_proto_files:/app/oppia/extensions/classifiers/proto
```

Comment:
Ditto*

Reply:
some proto files are also generated within `extensions/classifiers/proto`, and they are being imported so they must be present

Reply:
but do we need this in frontend containers? Also, why are we generating it during initiating fronted services?

Reply:
we are generating these proto file in backend stage
we need them as the ts/js files within extensions/classifiers/proto are imported in the other files

Reply:
I have two questions:

1. Are we generating these files in the image? If yes, then how is it conserved even after we bind the local repo to the container?
2. If it exists in the local/host repo do we need to create separate volumes for these files?